client/src/main/scala/org/zalando/nakadi/client/scala/ClientImpl.scala

```diff
@@ -155,18 +158,18 @@ class ClientImpl(connection: Connection, subscriber: SubscriptionHandler, charSe
   //#  HELPER METHODS  #
   //####################
 
-  private def logFutureEither[A, T](future: Future[Either[ClientError, T]]): Future[Either[ClientError, T]] = {
+  private def logFutureEither[TA, TB](future: Future[Either[ClientError, TB]]): Future[Either[ClientError, TB]] = {
     future recover {
       case e: Throwable =>
-        val msg = s"An unexpected error occured: ${e.getMessage}"
+        val msg = s"An(1) unexpected error occured: ${e.getMessage}"
```

Comment:
Because the Error message was displayed in different functionalities, this was hard to find out where the error actually occurred...